lib/bolt/plugin/module.rb

```diff
@@ -155,7 +155,17 @@ def process_params(task, opts)
         # handled previously. That may not always be the case so filter them
         # out now.
         meta, params = opts.partition { |key, _val| key.start_with?('_') }.map(&:to_h)
-        params = config.merge(params)
+
+        # Reject parameters from config that are not accepted by the task and
+        # merge in parameter defaults
+        params = if task.parameters
+                   config.slice(*task.parameters.keys)
+                         .merge(task.parameter_defaults)
```

Comment:
This order seem backward. `config` should override `parameter_defaults`, right?

Reply:
Yeeep. Updated!